upi/openstack/021_network.yaml

```diff
@@ -6,35 +6,29 @@
   tasks:
   - name: 'Create ACI containers node network'
     command:
-      cmd: "neutron net-create {{ os_aci_containers_network }} --apic:nested-domain-name openshift-domain --apic:nested-domain-type openshift --apic:nested_domain_infra_vlan {{ infra_vlan }} --apic:nested_domain_service_vlan {{ service_vlan }}"
-    when: os_networking_type == "CiscoACI"
+      cmd: "neutron net-create {{ os_aci_containers_network }} --apic:nested-domain-name openshift-domain --apic:nested-domain-type openshift --apic:nested_domain_infra_vlan {{ aci_cni['infra_vlan'] }} --apic:nested_domain_service_vlan {{ aci_cni['service_vlan'] }}"
 
   - name: 'Set the ACI containers cluster network tag'
     command:
       cmd: "openstack network set --tag {{ cluster_id_tag }} {{ os_aci_containers_network }}"
-    when: os_networking_type == "CiscoACI"
 
   - name: 'Create the ACI containers subnet'
     os_subnet:
       name: "{{ os_aci_containers_subnet }}"
       network_name: "{{ os_aci_containers_network }}"
       no_gateway_ip: yes
-      cidr: "{{ os_aci_containers_subnet_range }}"
-      allocation_pool_start: "{{ os_aci_containers_subnet_range | next_nth_usable(10) }}"
-      allocation_pool_end: "{{ os_aci_containers_subnet_range | ipaddr('last_usable') }}"
-    when: os_networking_type == "CiscoACI"
+      cidr: "{{ aci_cni['network_interfaces']['opflex']['subnet'] }}"
+      allocation_pool_start: "{{ aci_cni['network_interfaces']['opflex']['subnet'] | next_nth_usable(10) }}"
+      allocation_pool_end: "{{ aci_cni['network_interfaces']['opflex']['subnet'] | ipaddr('last_usable') }}"
 
   - name: 'Set the ACI containers cluster subnet tag'
     command:
       cmd: "openstack subnet set --tag {{ cluster_id_tag }} {{ os_aci_containers_subnet }}"
-    when: os_networking_type == "CiscoACI"
 
   - name: 'Set MTU for the ACI containers node  network'
```

Comment:
Though not part of this PR, this name seems incorrect, it should be "Set MTU for the ACI containers network", no?

Reply:
Correct, changed!